subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -52,171 +52,207 @@
                             <li><a href="https://guides.gradle.org/creating-build-scans/">Creating Build Scans</a></li>
-                            <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating from Maven</a></li>
+                            <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
                         </ul>
 
                         <h3 id="using-gradle-builds">Using Gradle Builds</h3>
                         <ul>
+                            <li><a href="/userguide/build_cache.html">Build Cache</a></li>
+                            <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scans</a></li>
                             <li><a href="/userguide/command_line_interface.html">Command-Line Interface</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
+                                <ul id="customizing-execution">
+                                    <li><a href="/userguide/build_environment.html">Configuring Build Environment</a></li>
+                                    <li><a href="/userguide/gradle_daemon.html">Configuring Gradle Daemon</a></li>
+                                    <li><a href="/userguide/init_scripts.html">Initialization Scripts</a></li>
+                                </ul>
+                            </li>
+                            <li><a href="/userguide/intro_multi_project_builds.html">Executing Multi-Project Builds</a></li>
                             <li><a href="/userguide/gradle_wrapper.html">Gradle Wrapper</a></li>
-                            <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scans</a></li>
-                            <li><a href="/userguide/build_environment.html">Build Environment</a></li>
-                            <li><a href="/userguide/init_scripts.html">Initialization Scripts</a></li>
-                            <li><a href="/userguide/intro_multi_project_builds.html">Multi-Project Builds</a></li>
-                            <li><a href="/userguide/build_cache.html">Build Cache</a></li>
-                            <li><a href="/userguide/composite_builds.html">Composite Builds</a></li>
-                            <li><a href="/userguide/continuous_build.html">Continuous Build</a></li>
-                            <li><a href="/userguide/gradle_daemon.html">Daemon</a></li>
                             <li><a href="/userguide/troubleshooting.html">Troubleshooting</a></li>
-                            <li><a href="/userguide/embedding.html">Embedding Gradle</a></li>
                         </ul>
 
-                        <h3 id="writing-gradle-builds">Authoring Gradle Builds</h3>
+                        <h3 id="tutorials">Project Tutorials</h3>
```

Comment:
Should we rather say "Language Tutorials". I am not quite sure what "Project" is supposed to mean.

Reply:
It's supposed to mean that you build a project. Not sure how to tersely make that clear.

Reply:
Alright, I wrote `Project-Based Tutorials` here